bc_obps/registration/api/operator.py

```diff
@@ -46,6 +46,7 @@ def get_operator_by_legal_name_or_cra(
     response={200: List[OperatorOut], codes_4xx: Message, codes_5xx: Message},
     url_name="get_operators_by_legal_name",
 )
+@authorize(AppRole.get_all_app_roles(), UserOperator.get_all_industry_user_operator_roles(), False)
```

Comment:
Sepehr's got a PR that narrows down the return type to only the info a non-approved user can see

Reply:
I think we have to let this endpoint open to unapproved users otherwise they can't search for operators to request access!